docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -228,7 +228,16 @@ The above examples use the timestamp ``2001-08-22 03:04:05.321`` as the input.
 
 .. function:: date_trunc(unit, x) -> [same as input]
 
-    Returns ``x`` truncated to ``unit``.
+    Returns ``x`` truncated to ``unit``::
+
+        SELECT date_trunc('day' , TIMESTAMP '2022-10-20 05:10:00')
+        -- 2022-10-20 01:00:00.000
```

Comment:
Shouldn't the time be `00:00:00.000`, here and in the example for `month`?

Reply:
Yes. Updated it.